### DISCUSSION TITLE
How will we ensure the website’s performance remains consistent as new features are added?

### DISCUSSION THREAD

**Discussion by shubhagarwal1** [GDSC-RCCIIT/gdg-website#341] [Q&A] [answered]
With our roadmap expanding and more interactive elements being introduced, it’s essential we discuss strategies for load handling, optimization, and testing. Are there specific performance benchmarks or monitoring tools we’re considering to maintain a smooth user experience?

**Comment by Shariq2003** [comment]
We have to ensure optimisation by making a build for every short period (at minimum one week) after adding features.
That make sure of optimization by ESLint

**Comment by shubhagarwal1** [comment]
Weekly builds sound great for steady optimization, but how do we handle regression testing for previously optimized features?

**Reply by Shariq2003** [reply, marked as answer]
By testing before merging of that feature.
To handle regression testing for previously optimized features, we can implement automated testing workflows

**Reply by shubhagarwal1** [reply]
sounds good, we can add automated workflows for the testing previous features, this way our code will remain tight and contact, also will make it robust while pushing the things to the prod.
well thankyou for the suggestion, this sounds satisfying
